src/Logging/Logging.Testing/src/Microsoft.Extensions.Logging.Testing.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Helpers for writing tests that use Microsoft.Extensions.Logging. Contains null implementations of the abstractions that do nothing, as well as test implementations that are observable.</Description>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
This breaks .NET Framework tests. I was trying this to fix builds but this approach won't work. To summarize the issue:

1. xunit.extensibility.execution has a netstandard1.1 assembly called xunit.execution.dotnet.dll and a net452 assembly called xunit.execution.desktop.dll
2. We want to produce Microsoft.AspNetCore.Testing (MAT) reference assemblies to ensure all users of it reference only 3.0.0.0 ref assemblies.
3. MAT references xunit.extensibility.execution and only the netstandard2.0 reference assembly is generated. There's no ref assembly generated for net461.
4. Microsoft.Extensions.Logging.Testing (MELT) references MAT and can't build property for net461 since there's a transitive reference to xunit.execution.dotnet.dll via MAT's netstandard2.0 reference assembly. xunit.execution.dotnet.dll can't be resolved since MELT brings in xunit.execution.desktop.dll instead.

Is there a way to resolve this conflict @dougbu @nguerrera? Since there are no changes here I think we might be able to use the released 3.0.0 MELT? It's just a testing package anyway.

Reply:
Why does this project need to change? Why not generate a separate .NET 4.6.1 ref/ assembly?

Reply:
Ah I thought we can only generate ref assemblies for .NET Core. Turns out we can generate ref assemblies for .NET Framework as well. Looks like that works.